src/System.IO.Pipelines/System/IO/Pipelines/ReadCursor.cs

```diff
@@ -1,14 +1,16 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Diagnostics;
 using System.Runtime.CompilerServices;
 using System.Text;
 
 namespace System.IO.Pipelines
 {
+    [DebuggerDisplay("{Segment}[{Index}]")]
     public readonly struct ReadCursor : IEquatable<ReadCursor>
```

Comment:
Can we rename it to Position?

Reply:
We are getting there. Want to keep PR size reasonable.

Reply:
ok